src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -117,15 +118,11 @@ public RobotMap() {
 		Component.leftWheelEncoder.setReverseDirection(true);
 		Component.leftWheel = new PositionEncodedMotor("leftWheel", new AccelerationCap(Component.pdp), new CustomPIDController(Component.leftWheelEncoder), new VictorSP(Port.PWM.leftDriveMotor));
 		Component.leftWheel.disablePID(); // TODO add encoders
-		// Component.leftWheel.setInverted(true);
 		Component.rightWheelEncoder = new CANEncoder(Port.CAN.rightEncoder);
 		Component.rightWheel = new PositionEncodedMotor("rightWheel", new AccelerationCap(Component.pdp), new CustomPIDController(Component.rightWheelEncoder), new VictorSP(Port.PWM.rightDriveMotor));
 		Component.rightWheel.disablePID(); // TODO add encoders
-		// Component.rightWheel.setInverted(false);
-		// Ball Intake
-		Component.ballIntake = new BallInnie(new VictorSP(Port.PWM.ballInnie));
-		// Ball Dumper
-		Component.ballDumper = new Dump(new VictorSP(Port.PWM.vomitElevator), new VictorSP(Port.PWM.vomitOuttakeRoller));
+		// Ball-Intake-Outtake
+		Component.ballIO = new BallIO(new VictorSP(Port.PWM.bioTopMotor), new VictorSP(Port.PWM.bioLeftMotor), new VictorSP(Port.PWM.bioMainMotor), new DoubleSolenoid(Port.Pneumatics.bioShifterDown, Port.Pneumatics.bioShifterUp));
```

Comment:
Wrap each `VictorSP` in a `Motor` -- look at `leftFlywheelMotor` for an example. (you can break it out into multiple lines)